packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -480,7 +480,7 @@ class _SummaryCardState extends State<SummaryCard> {
               },
               child: SmoothCard(
                 margin: EdgeInsets.zero,
-                color: const Color(0xfff5f6fa),
+                color: Theme.of(context).colorScheme.primary,
```

Comment:
As said, we already talked about this [here](https://github.com/openfoodfacts/smooth-app/pull/1249) but I have to say the primary color looks way too prominent in my opinion. What about completely removing any color for it and using the standart SmoothCard color

Reply:
Hi @monsieurtanuki ,
Initially, I tried secondary color but @teolemon suggested to go with primary color. You can find the thread [here](https://github.com/openfoodfacts/smooth-app/issues/1321).

Reply:
This is Robotoff, and I'd be tempted to keep the "Robotoff blue" across platforms and versions. @M123-dev @monsieurtanuki @bhattabhi013

Reply:
> As said, we already talked about this [here](https://github.com/openfoodfacts/smooth-app/pull/1249) but I have to say the primary color looks way too prominent in my opinion. What about completely removing any color for it and using the standart SmoothCard color

I tried doing it this way, here is the outcome:
<img width="245" alt="image" src="https://user-images.githubusercontent.com/47862474/160117355-9b44a140-ea29-4964-824d-ca1db122b3f6.png">

Reply:
I have no special interest in specific colors, I was just pointing out that the flutter standards provide 2 main colors. That should be enough for most apps, but if smoothie needs additional colors, why not?